src/test/scala/com/microsoft/hyperspace/index/IndexCollectionManagerTest.scala

```diff
@@ -134,6 +135,6 @@ class IndexCollectionManagerTest extends SparkFunSuite with SparkInvolvedSuite {
 
   test("refresh() throws exception if index is not found") {
     when(mockFileSystem.exists(new Path(indexSystemPath, "idx4"))).thenReturn(false)
-    intercept[HyperspaceException](indexCollectionManager.refresh("idx4"))
+    intercept[HyperspaceException](indexCollectionManager.refresh("idx4", REFRESH_MODE_FULL))
```

Comment:
What's the expected behavior for incremental?

Reply:
thanks @imback82 , added test